### PR TITLE
chore: remove some code related to multi currency

### PR DESCRIPTION
**Pull request by nicolasburtey**
this code will be reimplemented in a subsequent PR

### CODE REVIEW

File: test/integration/services/ledger/transaction.__spec__.ts [hunk 1]
```diff
@@ -501,30 +501,30 @@ describe("rebalancePortfolio", () => {
...
+    // wallet.user.currencies = _5050meta.currencies
+    // const error = wallet.user.validateSync()
+    // expect(error).toBeFalsy()
 
-    await ledger.rebalancePortfolio({
-      description: "rebalancePortfolio",
-      metadata: { type: "user_rebalance", pending: false },
-      wallet,
-    })
+    // await ledger.rebalancePortfolio({
```
Comment (bodymindarts) [inline test/integration/services/ledger/transaction.__spec__.ts:508]: Why are you commenting this code? If you ever need to look it up again its already in the git history and you can just check out this file before it was deleted. No need to keep it or?

**Review by bodymindarts — APPROVED** [review]
Approving but think we should delete all the commented code - its still in the git history.

File: test/integration/services/ledger/transaction.__spec__.ts (hunk 1 above)
Comment (nicolasburtey) [reply test/integration/services/ledger/transaction.__spec__.ts:508]: the tests will be re-implemented shortly with the new code infra. so keep this code around is saving time, versus trying to find back where it's from the git history.

**Comment by nicolasburtey** [comment]
merging following rebase

**Pull request merged**